docs/developer-docs/smart-contracts/overview.mdx

```diff
@@ -0,0 +1,46 @@
+# Overview
+
+## Smart contracts on ICP
+
+The smart contracts running on ICP are a powerful evolution of traditional smart contracts. ICP smart contracts are called canisters and they are computational units that combine both code and data. Canisters can contain any arbitrary code or data, from serving web pages to creating a secure messaging app or implementing a decentralized token exchange.
+
+<div class="text--center">
+<img src="/img/docs/canister.png" alt="Canister" width="400"/>
+</div>
+
+Canisters have special properties that allow developers to build scalable Web3 services. To better understand these properties, one has to consider canisters from different perspectives to form a complete picture of their capabilities.
+
+An Ethereum developer might think of canisters as smart contracts, whereas someone with an academic background might make associations with actors and the [actor model](https://en.wikipedia.org/wiki/Actor_model). 
+
+A systems engineer will likely consider operating systems processes, whereas  a virtual machine expert might think about WebAssembly module instances. 
+
+All of these associations are partial, but they are correct. Together, they form a comprehensive picture of what a canister is. Let’s look at them one by one. For a more detailed description you can watch [this](https://www.youtube.com/watch?v=LKpGuBOXxtQ).
+
+## Canisters as smart contracts
+
+Canisters are much like a smart contract in that their execution is governed by a secure protocol; in this case, the ICP protocol. Canisters running on ICP are tamper-proof, since their state can only be modified through messages that are executed on-chain. A canister’s state can  be audited and cryptographically verified using ICP’s chain-key cryptography.
```

Comment:
```suggestion
Canisters are much like a smart contract in that their execution is governed by a secure protocol; in this case, the ICP protocol. Canisters running on ICP are tamper-proof, since their state can only be modified through messages that are executed on-chain. A canister’s state can be audited and cryptographically verified using ICP’s chain-key cryptography.
```